samples/go/src/utils/response.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+)
+
+func Respond(w http.ResponseWriter, message string) {
+	w.Header().Add("Content-Type", "application/json")
+	io.WriteString(w, fmt.Sprintf(`{"message": "%s"}`, message))
```

Comment:
Echo also comes with `c.JSON(...)` which is really helpful, and no need to every single go project reimplement it.